_site/index.html

```diff
@@ -85,39 +84,39 @@ <h2>A multi-cluster Control Plane for defining, configuring and deploying Ingres
+      <p>Explore the wide range of features that Kuadrant offers to enable and empower <b>cluster operators</b> and <b>application developers</b> to work together to securely connect users with their services. Our goal is to empower cluster administrators to deploy and manage gateways at scale to provide application connectivity and give them the confidence to allow application developers to self service knowing that endpoints exposed via these gateways are compliant, secure and resilient. All of this is underpinned by powerful new ingress standard <a href="https://gateway-api.sigs.k8s.io/">Gateway API.</a> and the concepts behind <a href="https://gateway-api.sigs.k8s.io/references/policy-attachment/">policy attachment.</a>
     </div>
 
     <div class="row">
       <div class="col-xl-3 col-md-6 d-flex align-items-stretch" data-aos="zoom-in" data-aos-delay="100">
         <div class="icon-box">
           <div class="icon"><i class="bx bx-globe"></i></div>
-          <h4><a href="">Multi-Cluster Ingress with DNS</a></h4>
-          <p>Establish seamless connectivity and advanced DNS management for workloads in multi-cluster deployment models with Kuadrant.</p>
+          <h4><a href="">Multi-Cluster Gateways connectivity and Load Balancing</a></h4>
+          <p>Connect users to and balance traffic across your gateway instances using the kuadrant DNSPolicy API that supports advanced strategies such as GEO and Weighted without needing to manage the individual records and integrates with multiple DNS Providers.</p>
         </div>
       </div>
 
       <div class="col-xl-3 col-md-6 d-flex align-items-stretch mt-4 mt-md-0" data-aos="zoom-in" data-aos-delay="200">
         <div class="icon-box">
           <div class="icon"><i class="bx bx-lock"></i></div>
-          <h4><a href="">Robust Access Management</a></h4>
-          <p>Secure your services with Kuadrant's powerful <a href="https://github.com/Kuadrant/limitador">rate limiting</a> and <a href="https://github.com/Kuadrant/authorino">authentication/authorization</a>, ensuring safe and policy-controlled access.</p>
+          <h4><a href="">Secure Access</a></h4>
+          <p>Secure your Gateways with automatic ACME based TLS integration. Protect your Gateways and HTTPRoutes with Kuadrant's powerful policy based APIs for rate limiting and auth that integrate directly with the Gateway.</p>
         </div>
       </div>
 
       <div class="col-xl-3 col-md-6 d-flex align-items-stretch mt-4 mt-xl-0" data-aos="zoom-in" data-aos-delay="300">
         <div class="icon-box">
           <div class="icon"><i class="bx bx-shield"></i></div>
-          <h4><a href="">Advanced Security Controls</a></h4>
-          <p>Protect your applications with Kuadrant's comprehensive security features, encompassing adaptive, multi-cluster rate limiting and sophisticated authentication/authorization capabilities.</p>
+          <h4><a href="">Seamless E/W connectivity integration</a></h4>
```

Comment:
"E/W" -> "East-West"

"Connectivity Integration" and "Connectivity and Integration"